mmdet/models/backbones/detectors_resnet.py

```diff
@@ -230,13 +227,11 @@ def __init__(self,
                  stage_with_sac=(False, False, False, False),
                  rfp_inplanes=None,
                  output_img=False,
-                 pretrained=None,
```

Comment:
for backward compatibility, please do not delete kwarg pretrained. 
in __init__add 
```
        if pretrained is not None:
            warnings.warn('DeprecationWarning: pretrained is a deprecated \
                key, please consider using init_cfg')
            self.init_cfg = dict(type='Pretrained', checkpoint=pretrained)

Reply:
We still need to keep back-compatibility for C4 models